src/components/MAPI/clusters/CreateCluster/schemaUtils.ts

```diff
@@ -0,0 +1,173 @@
+import { GenericObjectType, UiSchema } from '@rjsf/utils';
+import { generateUID } from 'MAPI/utils';
+import { compare } from 'utils/semver';
+
+import ClusterNameWidget from './ClusterNameWidget';
+
+const uiSchemaProviderAWS: Record<string, GenericObjectType> = {
+  v0: {
+    'ui:options': {
+      order: [
+        'clusterName',
+        'clusterDescription',
+        'organization',
+        'controlPlane',
+        '*',
+      ],
+    },
+    clusterName: {
+      'ui:options': {
+        widget: ClusterNameWidget,
+      },
+    },
+  },
+};
+
+const uiSchemaProviderAzure: Record<string, GenericObjectType> = {
+  v0: {
+    'ui:options': {
+      order: [
+        'clusterName',
+        'clusterDescription',
+        'organization',
+        'controlPlane',
+        '*',
+      ],
+    },
+    clusterName: {
+      'ui:options': {
+        widget: ClusterNameWidget,
+      },
+    },
+  },
+};
+
+const uiSchemaProviderCloudDirector: Record<string, GenericObjectType> = {
+  v0: {
+    'ui:options': {
+      order: [
+        'clusterDescription',
+        'cluster',
+        'organization',
+        'controlPlane',
+        '*',
+      ],
+    },
+    clusterName: {
```

Comment:
Do we need `clusterName` here? I don't see it in the CloudDirector schema.

Reply:
No we don't - missed this one